projects/kit/components/input-date/multi-date/input-date-multi.template.html

```diff
@@ -0,0 +1,84 @@
+<tui-hosted-dropdown
+    class="t-hosted"
+    [canOpen]="canOpen"
+    [content]="dropdown"
+    [open]="open && canOpen"
+    (openChange)="onOpenChange($event)"
+>
+    <tui-input-tag
+        *tuiLet="filler$ | async as filler"
+        #inputTag
+        automation-id="tui-input-date-tag__textfield"
+        class="t-input"
+        [disabled]="disabled"
+        [disabledItemHandler]="disabledItemHandler | tuiMapper: disabledItemHandlerWrapper"
+        [editable]="editable"
+        [expandable]="expandable"
+        [inputHidden]="!editable"
+        [maskito]="optionsMaskito"
+        [nativeId]="nativeId"
+        [ngModel]="value"
+        [placeholder]="placeholder"
+        [pseudoFocus]="pseudoFocus"
+        [pseudoHover]="pseudoHover"
+        [pseudoInvalid]="computedInvalid"
+        [readOnly]="readOnly"
+        [removable]="!nativeDropdownMode"
+        [rows]="rows"
+        [tagValidator]="tagValidator"
+        [tuiTextfieldFiller]="getComputedFiller(filler || '')"
+        [tuiTextfieldIcon]="calendarIcon && iconContent"
+        (keydown.enter)="onEnter(inputTag.search)"
+        (ngModelChange)="onValueChange($event)"
+    >
+        <ng-content></ng-content>
+        <ng-content
+            ngProjectAs="input"
+            select="input"
+        ></ng-content>
+    </tui-input-tag>
+
+    <ng-template #iconContent>
+        <tui-svg
+            *polymorpheusOutlet="calendarIcon as src; context: {$implicit: size}"
+            appearance="icon"
+            automation-id="tui-input-date-range__icon"
+            tuiWrapper
+            [class.t-icon]="!computedDisabled"
+            [src]="src"
+            (click)="onIconClick()"
+        ></tui-svg>
+        <input
+            *ngIf="isMobile && nativePicker"
```

Comment:
Haven't we dropped native input for the time being?

Reply:
fixed